lib/portlayer/metrics/container.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
I think this is likely to break down as the different portlayer components providing metrics related to their area of concern, rather than a separate metrics package handling all of it.
For example, consider the case where NSX manager has the network usage data, not VC.

Don't change this for now, but please be aware we'll likely have to shuffle in the short/medium future.